.eslintrc.json

```diff
@@ -0,0 +1,298 @@
+{
+  "plugins": ["@typescript-eslint"],
+  "parser": "@typescript-eslint/parser",
+  "env": {
+    "es2021": true,
+    "node": true
+  },
+  "extends": [
```

Comment:
Don't really want to maintain a eslint config and keep it up to date, since [currently xo(-space)](https://github.com/obs-websocket-community-projects/obs-websocket-js/blob/47d74181694dd8d53842ee3052f7f9f470f763df/package.json#L76) is being used why not add [xo-typescript](https://github.com/xojs/eslint-config-xo-typescript#standalone-usage) to it?

Reply:
Sure